### PR TITLE
Get order by Id

### PR DESCRIPTION
**Pull request by dashakhan**
Added test for Get order by Id

### CODE REVIEW

File: test/orders/orderGetById.spec.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    let orderId, orderList = [], getOrder, clientId, vendorId, serviceId
+  before(async() =>{
+    clientId = (await clientHelper.createClient(clientHelper.clientData)).body.payload;
+    vendorId = (await vendorHelper.createVendor(vendorHelper.vendorData)).body.payload
+    serviceId = (await serviceHelper.createService(vendorId)).body.payload;
+  })
+  after(async () =>{
+    await vendorHelper.deleteVendor(vendorId);
+    await clientHelper.deleteClient(clientId);
+    await vendorHelper.deleteVendor(vendorId);
```
Comment (dumpdusty) [inline test/orders/orderGetById.spec.js]: guess it should be `deleteService`

File: test/orders/orderGetById.spec.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    it('verify response status', () => {
+      expect(getOrder.statusCode).to.eq(200)
+    })
+
+    it('verify response message', () => {
+      expect(getOrder.body.message).to.eq('Get Order by id ok')
+    })
+
+    it('verify response success - true', () => {
```
Comment (dumpdusty) [inline test/orders/orderGetById.spec.js:37]: guess better to verify if returned `orderId` is matches provided one

File: test/orders/orderGetById.spec.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      })
+
+      it('verify response success - false', () => {
+        expect(getOrder.body.success).to.eq(false)
+      })
+
+      it('verify response body structure', () => {
+        expect(getOrder.body).to.have.all.keys( 'message', 'success', 'fail', 'silent');
+      })
+    })
```
Comment (dumpdusty) [inline test/orders/orderGetById.spec.js:122]: can you please add the test for non-existing order (using deleted order Id)

**Review by dumpdusty — COMMENTED** [review]
@dashakhan some minor things in comments may need to be addressed

**Comment by dashakhan** [comment]
> @dashakhan some minor things in comments may need to be addressed

On it!

**Review by dumpdusty — DISMISSED** [review]
Thanks for fixing! All good so far!

File: test/orders/orderGetById.spec.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    describe('GET ORDER BY ID - WITH INVALID ID', () =>{
+      before(async() =>{
+        orderId = (await orderHelper.createOrder(orderHelper.orderData(clientId, serviceId))).body.payload
+        orderList.push(orderId + "+++INVALID+++")
```
Comment (RafailFerre) [inline test/orders/orderGetById.spec.js]: and how will this order be deleted later?

File: test/orders/orderGetById.spec.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        getOrder = await orderHelper.getOrder('')
```
Comment (RafailFerre) [inline test/orders/orderGetById.spec.js]: it's like without id, but not invalid

**Review by RafailFerre — APPROVED** [review]

**Review by RafailFerre — APPROVED** [review]

**Review by dumpdusty — APPROVED** [review]
LGTM!

**Pull request merged**